content/guides/integrations/add-wandb-to-any-library.md

```diff
@@ -76,46 +76,58 @@ dev = [
 
 ### User Login
 
-There are a few ways for your users to log in to W&B:
+#### Create an API key
+
+An API key authenticates a client or machine to W&B. You can generate an API key from your user profile.
+
+{{% alert %}}
+For a more streamlined approach, you can generate an API key by going directly to [https://wandb.ai/authorize](https://wandb.ai/authorize). Copy the displayed API key and save it in a secure location such as a password manager.
+{{% /alert %}}
+
+1. Click your user profile icon in the upper right corner.
+1. Select **User Settings**, then scroll to the **API Keys** section.
+1. Click **Reveal**. Copy the displayed API key. To hide the API key, reload the page.
+
+#### Install the `wandb` library and log in
+
+To install the `wandb` library locally and log in:
 
 {{< tabpane text=true >}}
+{{% tab header="Command Line" value="cli" %}}
 
-{{% tab header="Bash" value="bash" %}}
-Log into W&B with a bash command in a terminal:
+1. Set the `WANDB_API_KEY` [environment variable]({{< relref "/guides/models/track/environment-variables.md" >}}) to your API key.
 
-```bash
-wandb login $MY_WANDB_KEY
-```
-{{% /tab %}}
+    ```bash
+    export WANDB_API_KEY=<your_api_key>
+    ```
 
-{{% tab header="Notebook" value="notebook" %}}
-If they're in a Jupyter or Colab notebook, log into W&B like so:
+1. Install the `wandb` library and log in.
 
-```python
-import wandb
-wandb.login()
-```
-{{% /tab %}}
 
-{{% tab header="Environment Variable" value="environment" %}}
-Set a [W&B environment variable]({{< relref "/guides/models/track/environment-variables.md" >}}) for the API key:
 
-```bash
-export WANDB_API_KEY=$YOUR_API_KEY
-```
+    ```shell
+    pip install wandb
 
-or
+    wandb login
+    ```
 
-```python
-os.environ['WANDB_API_KEY'] = "abc123..."
-```
 {{% /tab %}}
 
+{{% tab header="Python" value="python" %}}
```

Comment:
The example below could be Python, but you'd need to remove the `!` in the pip install statement.

Or, rename "Python" to "Python notebook", "Juptyer Notebook", "Google Colab", etc.

Reply:
This applies to this example, and all the other markdown files in this PR.